src/components/Alert/README.md

```diff
@@ -171,13 +171,13 @@ React.createElement(() => {
 ## Forwarding HTML Attributes
 
 In addition to the options below in the [component's API](#api) section, you
-can specify [React synthetic events] or **any HTML attribute you like.** All
-attributes that don't interfere with the API are forwarded to the root `<div>`
-HTML element. This enables making the component interactive and helps to improve
+can specify **any HTML attribute you like.** All attributes that don't interfere
+with the API of the React component are forwarded to the root `<div>` HTML
+element. This enables making the component interactive and helps to improve
 its accessibility.
 
-👉 Refer to the MDN reference for the full list of supported attributes of the
-[div] element.
+👉 For the full list of supported attributes refer to [div HTML element attributes]
```

Comment:
I am aware it wasn't there before, but I wish we could polish the formatting like this:

```suggestion
👉 For the full list of supported attributes refer to the [`div` HTML element attributes]
```

Does it work?

Reply:
> Does it work?

No, I have tested it bedore.

At least I made the links open in new tab.
@bedrich-schindler @adamkudrna can I copy this change to other README files?

Reply:
On fact, there is a way. I did change it as suggested.